src/common/imageio_avif.c

```diff
@@ -230,10 +217,43 @@ int dt_imageio_avif_read_profile(const char *filename, uint8_t **out, dt_colorsp
     cicp->color_primaries = avif_image.colorPrimaries;
     cicp->transfer_characteristics = avif_image.transferCharacteristics;
     cicp->matrix_coefficients = avif_image.matrixCoefficients;
+
+    /* fix up mistagged legacy AVIFs */
+    if(avif_image.colorPrimaries == AVIF_COLOR_PRIMARIES_BT709)
+    {
+      bool over = false;
```

Comment:
sorry did not spot that before, you need to use gboolean (bool will break on some platforms) and FALSE.

Reply:
Done

Reply:
I don't see any change! A missing push?

Reply:
Argh, forced pushed an empty commit, forgot to stage 🤦